tests/e2e/support/objects/app-files/resource/actions.ts

```diff
@@ -135,15 +136,14 @@ export const createSpaceFromSelection = async ({
   })
   await page.locator(util.format(resourceNameSelector, resources[0])).click({ button: 'right' })
 
-  await page.locator('text=Create Space from selection').first().click()
-  await page.locator('.oc-text-input').first().fill(spaceName)
+  await page.locator(createSpaceFromResourceAction).first().click()
+  await page.locator(resourceNameInput).first().fill(spaceName)
```

Comment:
do we need `first()` here?
![Uploading Screenshot 2023-04-05 at 18.08.20.png…]()